Sources/StreamVideo/WebRTC/StreamVideoCaptureHandler.swift

```diff
@@ -57,26 +64,40 @@ class StreamVideoCaptureHandler: NSObject, RTCVideoCapturerDelegate {
         for buffer: RTCCVPixelBuffer,
         frame: RTCVideoFrame
     ) -> RTCVideoFrame {
-        var rotation = RTCVideoRotation._0
+        var rotation = frame.rotation
         switch sceneOrientation {
         case .portrait:
             rotation = ._90
         case .portraitUpsideDown:
             rotation = ._270
-        case .landscapeRight:
-            rotation = currentCameraPosition == .front ? ._180 : ._0
         case .landscapeLeft:
             rotation = currentCameraPosition == .front ? ._0 : ._180
+        case .landscapeRight:
+            rotation = currentCameraPosition == .front ? ._180 : ._0
         default:
-            rotation = ._90
```

Comment:
In the case we don't recognise the device's orientation we don't make any changes to the frame's rotation. I don't see how that can be wrong but setting it in ._90 degrees always isn't. If you also have a link with the code from WebRTC that may help me understand more the why

Reply:
Best to unpack the WebRTC framework and check the `RTCameraVideoCapturer`. I don't remember exactly what was the issue, I think it was because of the `unknown` interface orientation.
We haven't had any problems with this code on many devices for the last 8-9 months.

Reply:
You are right. I'll revert the changes to align with the WebRTC implementation